Factory.Cargo/CargoFactoryApi.cs

```diff
@@ -0,0 +1,36 @@
+using Factory.Cargo.Factory;
+using Factory.Cargo.TransportationFactory;
+
+namespace Factory.Cargo;
+
+public class CargoFactoryApi
```

Comment:
why did you add API? 
it is  a bad naming selection.

Reply:
@mohammadKarimi
I was looking for some distinguish between package class and last practice class.
I have changed the name from Api to Service.
Please checkout [024ad3e].